### ISSUE TITLE
npt test is broken on ppc64le

### ISSUE THREAD

**Issue by junghans** [espressomd/espresso#2468]
Building 4.0.1 on OpenSuse:
```
[ 1489s]  82/113 Test  #82: npt .....................................***Failed   17.54 sec
[ 1489s] 1.9911326746382318 0.21668762545656178
[ 1489s] F
[ 1489s] ======================================================================
[ 1489s] FAIL: test_npt (__main__.NPTintegrator)
[ 1489s] ----------------------------------------------------------------------
[ 1489s] Traceback (most recent call last):
[ 1489s]   File "/home/abuild/rpmbuild/BUILD/espresso-4.0.1/build/testsuite/python/npt.py", line 79, in test_npt
[ 1489s]     self.assertAlmostEqual(0.2, compressibility, delta=0.013)
[ 1489s] AssertionError: 0.2 != 0.21668762545656178 within 0.013 delta
[ 1489s] 
[ 1489s] ----------------------------------------------------------------------
[ 1489s] Ran 1 test in 14.682s
[ 1489s] 
[ 1489s] FAILED (failures=1)
[ 1489s] -------------------------------------------------------
[ 1489s] Primary job  terminated normally, but 1 process returned
[ 1489s] a non-zero exit code.. Per user-direction, the job has been aborted.
[ 1489s] -------------------------------------------------------
[ 1489s] --------------------------------------------------------------------------
[ 1489s] mpiexec detected that one or more processes exited with non-zero status, thus causing
[ 1489s] the job to be terminated. The first process to do so was:
[ 1489s] 
[ 1489s]   Process name: [[57741,1],0]
[ 1489s]   Exit code:    1
[ 1489s] --------------------------------------------------------------------------
[ 1489s] 
```

**Comment by junghans** [comment]
Same on ppc64

**Comment by junghans** [comment]
I can reproduce this on ppc64* and **aarch64** on Fedora 28 and 29, but not Fedora 30.

**Comment by junghans** [comment]
And on i686 (Fedora 28 & 29) `npt` is just hanging, on Fedora 30 it runs fine.

**Comment by junghans** [comment]
The i686 might be related to boost-1.66, even though that wouldn't explain why it succeeds  on f30.

**Comment by mkuron** [comment]
CMake should just show an error on i686, saying that we require Boost 1.67.0.

**Comment by junghans** [comment]
> CMake should just show an error on i686, saying that we require Boost 1.67.0.

Due to other issues in boost, Fedora still only comes with 1.66.0, so I had to patch that on i686, however that does explain the issue on the other 64-bit platforms.

**Comment by junghans** [comment]
Fedora 30 comes with gcc-9.0.1 while Fedora 29 is still gcc-8, so that might be the issue...

**Comment by mkuron** [comment]
We have GCC 8.2 in our Ubuntu images, so that's probably not it. I don't really have an explanation for what looks like a numerical precision issue.

**Comment by junghans** [comment]
We had a similar issue with votca-csg recently, however if you have a Fedora VM it is not that hard to do a cross-arch build:
https://github.com/votca/votca/blob/master/share/doc/DEVELOPERS_GUIDE.md#failed-release-builds
you can use the spec file from here:
https://src.fedoraproject.org/rpms/espresso/blob/master/f/espresso.spec
and of source you have to re-enable the test here:
https://src.fedoraproject.org/rpms/espresso/blob/master/f/espresso.spec#_188

**Comment by junghans** [comment]
Everything fixed in F30 with boost-1.69.0!
